tests/app/templating/test_schema_context.py

```diff
@@ -233,3 +233,112 @@ def test_respondent_display_name_is_ru_name_as_when_trading_as_empty(self):
         schema_context = build_schema_context(metadata, {}, self.answer_store)
 
         self.assertEqual(schema_context['respondent']['trad_as_or_ru_name'], metadata['ru_name'])
+
```

Comment:
Do we really need to test json_safe on every one(ru_name, trade_as etc) surely we just need to test json_safe with one example